helpers/src/helpers.js

```diff
@@ -0,0 +1,115 @@
+const data = require('./data')
```

Comment:
I don't like the fact we have 'helpers/src/helpers.js', knowing we also have 'helpers/index.js'

Reply:
I just kept the structure from oz/gsn-helpers, so probably need to rename the inner dir.